### PR TITLE
bazel: pretty print is all but pretty

### PR DESCRIPTION
**Pull request by haampie**
tensorflow builds are completely unreadable because the entirety of envp / argv
is printed. `--verbose_failures` looks like it's enough to me.

### CODE REVIEW

File: var/spack/repos/builtin/packages/bazel/package.py [hunk 1]
```diff
@@ -221,13 +221,8 @@ def setup_build_environment(self, env):
             "--color=no --host_javabase=@local_jdk//:jdk"
             # Enable verbose output for failures
             " --verbose_failures"
-            # Ask bazel to explain what it's up to
-            # Needs a filename as argument
-            " --explain=explainlogfile.txt"
-            # Increase verbosity of explanation,
-            " --verbose_explanations"
             # Show (formatted) subcommands being executed
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/bazel/package.py]: This comment should be removed too

File: var/spack/repos/builtin/packages/py-tensorflow-hub/package.py [hunk 2]
```diff
@@ -49,8 +49,6 @@ def install(self, spec, prefix):
             "--jobs={0}".format(make_jobs),
             # Enable verbose output for failures
             "--verbose_failures",
-            # Show (formatted) subcommands being executed
-            "--subcommands=pretty_print",
             "--spawn_strategy=local",
             # Ask bazel to explain what it's up to
             # Needs a filename as argument
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/py-tensorflow-hub/package.py]: Did you want to remove these too for consistency? Same in other files.

**Review by adamjstewart — COMMENTED** [review]
I'm fine with this, the logs have always been illegible to me. However, @glennpj and @aweits have debugged far more bazel builds than me, so I would like them to chime in.

File: var/spack/repos/builtin/packages/py-tensorflow-hub/package.py (hunk 2 above)
Comment (haampie) [reply var/spack/repos/builtin/packages/py-tensorflow-hub/package.py]: Yeah, I think it's generating about 250MB of output, but didn't verify this...

**Review by adamjstewart — COMMENTED** [review]
Also `py-dm-tree`

**Comment by haampie** [comment]
I skipped py-dm-tree cause it looks very brittle with filter_file, and I didn't want to have my name show up in git blame there :p but ok, added that one too.

**Review by adamjstewart — APPROVED** [review]
Looks fine with me, will wait to hear from @glennpj and @aweits

**Review by glennpj — APPROVED** [review]
This looks fine to me.

**Review by tgamblin — APPROVED** [review]

**Comment by haampie** [comment]
@spackbot run pipeline

**Comment by aweits** [comment]
I'll look to see what the output looks like on builds with this off, but I suspect we'll want to turn this back on to have enough information in the logs/issues when the odd problems manifest. I'd rather not ask folks to have to rerun builds that fail with other options turned on when they open issues.

**Comment by haampie** [comment]
It doesn't drop `verbose_failures`, although I didn't check what that actually does.

**Comment by alecbcs** [comment]
@spackbot run pipeline

**Pull request merged**